shared/ssl.sh

```diff
@@ -25,6 +30,8 @@
 #   PUPPETSERVER_HOSTNAME  Hostname of Puppet Server CA, defaults to "puppet"
 #   SSLDIR                 Root directory to write files to, defaults to
 #                          "/etc/puppetlabs/puppet/ssl"
+#   DNS_ALT_NAMES          Space-separated string of DNS subject alternative
+#                          names, defaults to none
```

Comment:
Hm, I'm wondering if this will cause confusion since the `DNS_ALT_NAMES` variable in the puppetserver container is a comma-separated list. Seems like standardizing and doing whatever data processing is necessary would be good 🤷‍♀

Reply:
Yeah....I've since found that it's difficult (impossible?) to append to a space-separated list in code, because you need to use quotes in order to define it, but you don't want the quotes to turn the value into a single element.

So yeah....think I'm going to try and rewrite this to be comma-separated and make sure it's portable enough.

Reply:
Ok found a way to support that that works everywhere: `tr "," " "`

🎉